modules/bbp-core/src/main/resources/schemas/bbp/core/typedlabeledontologyterm/v0.1.0.json

```diff
@@ -42,10 +42,11 @@
         }
       ]
     },
+
     {
-      "@id": "this:BrainRegionOntologyTermShape",
+      "@id": "this:ObjectiveTypeOntologyTermShape",
```

Comment:
Why the this:BrainRegionOntologyTermShape shape is removed ?

Reply:
will correct this